src/styles/page-template/index.md.njk

```diff
@@ -16,11 +16,11 @@ This page template combines the boilerplate markup and [components](../../compon
 - the [skip link](../../components/skip-link), [header](../../components/header) and [footer](../../components/footer) components
 - the favicon, and other related theme icons
 
-In the examples provided, we show both HTML and Nunjucks.
+In the examples provided, we show both HTML and [Nunjucks](https://frontend.design-system.service.gov.uk/use-nunjucks-macros/).
```

Comment:
As we're linking to the nunjucks guidance in the Frontend docs from here, I think this makes it more important that we talk about both the page template (which is not a macro) and the macros in that guidance.

Reply:
I think this is fixed now, but let me know what you think.

The user journey as I see it is that:
- user lands on this page because they want to use the page template
- they're taken to Frontend docs here, which in a section now called 'set up Nunjucks and the page template' shows them how to set up Nunjucks (and refers them briefly back to this page for the template code)
- at the end of that section, they're brought back to this page for docs on changing the template (OR they continue in Frontend docs to the section on adding components macros)

Hope that makes sense!